test.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from dojo import Dojo
+
+class TestCreateRoom(unittest.TestCase):
+    """Test cases for creating room"""
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_adds_office_successfully(self):
+        """Tests if room of type office can be added successfully"""
+        office = self.dojo.create_room("office", "A04")
+        self.assertEqual(office, "Office added successfully")
+
+    def test_adds_living_space_successfully(self):
+        """Tests if room of type livingSpace can be added successfully"""
+        living_space =self.dojo.create_room("living_space", "Arusha")
+        self.assertEqual(living_space, "Living space added successfully")
```

Comment:
refer to earlier comment on checking the count and that the created room is a `Living space`.